assets/data/materials/boss-material/en.json

```diff
@@ -0,0 +1,32 @@
+{
+    "anemo": {
+        "slug": "hurricane-seed",
```

Comment:
Slug might not be the correct key for this, as we're pretty much using this as the unique identifier for all other entities. Would `id` be a more proper way to name this field (I would then change this across all files in this pull request)

Reply:
Yeah, I think using `id` would be better in this case.